lib/data/wrapper.js

```diff
@@ -170,6 +196,7 @@ const data = {
             return cb(null, defResp);
         }
         return client.healthcheck(log, (err, result) => {
+            // NEED TO UPDATE FOR POSSIBILITY OF MULTIPLE HEALTHCHECKS HERE!!
```

Comment:
this is part of the todo for the healthcheck in the gateway.  let's have a todo here and please handle in the subsequent pr.